int32 updown_state  #0x00无动作 0x01 PC控下降中 0x02 PC控下降完成 0x03 PC控上升中 0x04 PC控上升完成
int32 left_axischange_state  #0x00无动作 0x01 PC控减小中 0x02 PC控减小完成 0x03 PC控增大中 0x04 PC控增大完成
int32 right_axischange_state
int32 carhold_state  #0x00无动作 0x01自动取车中 0x02自动取车完成0x03自动放车中 0x04自动放车完成
int32 heartbeat


